### PR TITLE
bootstrap diabetes cleanup

### PR DESCRIPTION
**Pull request by dtzar**
Rename all remaining traces of the word "diabetes" in the code base.

### CODE REVIEW

File: bootstrap/README.md [hunk 1]
```diff
@@ -2,8 +2,10 @@
 
 To use this existing project structure and scripts for your new ML project, you can quickly get started from the existing repository,  bootstrap and create a template that works for your ML project. Bootstraping will prepare a similar directory structure for your project which includes renaming files and folders, deleting and cleaning up some directories and fixing imports and absolute path based on your project name. This will enable reusing various resources like pre-built pipelines and scripts for your new project.
 
-To bootstrap from the existing MLOpsPython repository clone this repository and run bootstrap.py script as below
+To bootstrap from the existing MLOpsPython repository clone this repository, ensure Python is installed locally, and run bootstrap.py script as below
```
Comment (tcare) [inline bootstrap/README.md]: Python 3.x?

File: bootstrap/bootstrap.py [hunk 2]
```diff
@@ -108,6 +76,40 @@ def validateargs(self):
             raise Exception("Project name should be 3 to 15 chars long")
 
 
+def replaceprojectname(project_dir, project_name, rename_name):
+    # Replace instances of rename_name within files with project_name
+    dirs = [r".env.example",
+            r".pipelines\azdo-base-pipeline.yml",
```
Comment (tcare) [inline bootstrap/bootstrap.py]: \ won't work for xplat, might be out of scope for this PR.

File: bootstrap/bootstrap.py [hunk 3]
```diff
@@ -57,38 +57,6 @@ def deletedir(self):
             os.system(
                 'rmdir /S /Q "{}"'.format(os.path.join(self._project_directory, dir)))  # NOQA: E501
```
Comment (tcare) [inline bootstrap/bootstrap.py]: rmdir won't work for xplat, probably out of scope for this PR.

File: bootstrap/README.md (hunk 1 above)
Comment (dtzar) [reply bootstrap/README.md]: we're only using `os, sys, argparse` which should technically work on 2.x or 3.x

File: bootstrap/bootstrap.py [hunk 4]
```diff
@@ -108,6 +76,40 @@ def validateargs(self):
...
+        fin.close()
+        fin = open(os.path.join(project_dir, file),
+                   "wt", encoding="utf8")
+        fin.write(data)
+        fin.close()
+
+
 def main(args):
     parser = argparse.ArgumentParser(description='New Template')
     parser.add_argument("--d", type=str,
```
Comment (tcare) [inline bootstrap/bootstrap.py]: We should also stick to the standard of -d or --directory, same for below

File: bootstrap/bootstrap.py [hunk 5]
```diff
@@ -108,6 +76,40 @@ def validateargs(self):
             raise Exception("Project name should be 3 to 15 chars long")
```
Comment (tcare) [inline bootstrap/bootstrap.py]: Now that we're replacing more things, I think our valid set of chars might be reduced. \w and _ are probably it.

**Pull request merged**